test/perf/hc-nested-list_perf.html

```diff
@@ -82,6 +82,11 @@
             it('should fetch a deeply nested items depth in <1ms', () =>
               expect(getPerformance(() =>
               element._getItemDepth(gChild || child || parent, parts)).avg).at.most(1));
+            it('should collapse open tree nodes in <1 ms', () => {
+              element.parts = parts;
+              element.parts['root'].forEach(item => item.open = true);
```

Comment:
This is best-case scenario, when performance testing you should always build off of the worst-case scenario, which means you need something like:
```
Object.keys(element.parts).forEach(key => {
  const items = element.parts[key];
  items[items.length - 1].open = true;
});
```
more importantly, this is something that gets changed as part of your test, so it should be in the second callback so it's executed before every test, otherwise it's only going to affect the first run.